requirements.txt

```diff
@@ -70,7 +70,7 @@ responses==0.9.0
 
 # A common package that holds the models deifnition and schemas that are used
 # accross different amundsen repositories.
-amundsen-common>=0.3.1
+amundsen-common>=0.3.1,<1.0
```

Comment:
@feng-tao, just curious how does this lower version work in requirements?

Reply:
I think so, do you still see the common in 0.3.1 version? could it be the case that you use pypi version which doesn't have it?

Reply:
I meant this part `<1.0`. what does this lower version do ?